tsconfig.json

```diff
@@ -24,6 +24,10 @@
     "types": ["react", "node", "jest"],
     "newLine": "LF",
     "pretty": true,
+    "typeRoots": ["./@types"],
+    "paths": {
+      "@images/*": ["assets/images/*"]
```

Comment:
I believe we can drop the `paths` property as we are using the `~` and it's already defined as an alias in the webpack config.